packages/server/graphql/mutations/addComment.ts

```diff
@@ -89,6 +91,44 @@ const addComment = {
       }
     }
 
+    const {entityMap} = JSON.parse(content)
+    const notificationsToAdd = getTypeFromEntityMap('MENTION', entityMap)
+      .filter((mentionedUserId) => {
+        if (mentionedUserId === viewerId) {
+          return false
+        }
+
+        if (discussion.discussionTopicType === 'teamPromptResponse') {
+          const {userId: responseUserId} = TeamMemberId.split(discussion.discussionTopicId)
+          if (responseUserId === mentionedUserId) {
+            // The mentioned user will already receive a 'RESPONSE_REPLIED' notification for this
+            // comment
+            return false
+          }
+        }
+
+        // :TODO: (jmtaber129): Consider limiting these to when the mentionee is *not* on the
+        // relevant page.
```

Comment:
+1 it'd be good to create a follow up task for that

Reply:
https://github.com/ParabolInc/parabol/issues/7627